common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_cli.h

```diff
@@ -458,18 +458,19 @@ class cACTION_CLI_CLIENT_BSS_STEER_REQUEST : public BaseClass
         uint32_t* m_disassoc_timer_ms = nullptr;
 };
 
-class cACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST : public BaseClass
+class cACTION_CLI_CLIENT_CHANNEL_LOAD_11K_REQUEST : public BaseClass
 {
     public:
-        cACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST(uint8_t* buff, size_t buff_len, bool parse = false);
-        explicit cACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST(std::shared_ptr<BaseClass> base, bool parse = false);
-        ~cACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST();
+        cACTION_CLI_CLIENT_CHANNEL_LOAD_11K_REQUEST(uint8_t* buff, size_t buff_len, bool parse = false);
```

Comment:
again - please provide the tlvf and the autogenerated files in the same commit - no added value in splitting them up

Reply:
fixed